pkg/download/README.md

```diff
@@ -0,0 +1,22 @@
+# Download configuration
+
+This feature allows you to download the configuration from a Dynatrace environment as Monaco files. You can use this feature to avoid starting from scratch when using Monaco.
+
+### steps
+1. Create an environment file. 
+2. Run the monaco command using the download flag ``-download``
```

Comment:
Generally we aim to document parameters as `--long-name` the way most CLI tools handle them, even if Go flags allows both/single `-`